app/views/pages/index.html.erb

```diff
@@ -50,7 +50,7 @@
       <article class="HomepageCategory">
         <%= link_to "/docs/integrations/source-control", class: "HomepageCategory__header" do %>
           <h3 class="HomepageCategory__heading">Source control</h3>
-          <p class="HomepageCategory__subtext">Trigger builds from wherever your code is stored</p>
+          <p class="HomepageCategory__subtext">Trigger builds from your code repo</p>
```

Comment:
style, optional: I'm not wild about "repo" as a shorthand on first mention

```suggestion
          <p class="HomepageCategory__subtext">Trigger builds from your repository</p>
```

Reply:
Nice! Accepted suggestion ✅